### ISSUE TITLE
Add links to how.bitshares.works after it's ready

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#953]
Places that I've thought of:
* README
* Wiki

**Comment by cedar-book** [comment]
@abitmore, I have moved all wiki contents in dev.bitshares.works repository. Some contents were "ideas" or "todos".  I put together those contents in this [page](https://github.com/bitshares/dev.bitshares.works/tree/master/core/knowledge_base#knowledge-base).  If you are okay, I can move links one by one. If you have links, want to wait to switch I won't do that.

**Comment by oxarbitrage** [comment]
with this https://github.com/bitshares/bitshares-core/issues/1031 done we can drive our attention here.

**Comment by oxarbitrage** [comment]
added link to dev portal at wiki https://github.com/bitshares/bitshares-core/wiki front page.

"New and updated developer information can be found at Bitshares Developer Portal"

**Comment by cedar-book** [comment]
Very beginning (in May), I started to use "how.bitshares.works" repository instead of "dev.bitshares.works" repository for the developers documentation.  
If I remember correctly, this request was "after BitShsares-Core repository's Readme and wiki's contents were moved to new doc location (in dev.bitshares.works), replace current wiki's link to the new content location."  

Currently, the dev.bitshares.works documents source files are RST format to build the html.  I have moved most of them, some of them are mixed with others or in Tutorials/FAQs. And if I could not find a good group section to add, I put all on a [wiki legacy page](https://dev.bitshares.works/en/master/knowledge_base/wiki_legacy/wiki_legacy_index.html#wiki-legacy-indx) in Knowledge Base section. 

I created this [contribution guide](https://github.com/bitshares/dev.bitshares.works/blob/master/CONTRIBUTING.md) (in July).  It will be updated.
